packages/main/src/types/ToolbarStyling.ts

```diff
@@ -0,0 +1,26 @@
+/**
+* Defines the design of ui5-toolbar background.
+ *
+ * @readonly
+ * @enum {string}
+ * @type {string}
+ * @public
+ * @author SAP SE
+ * @alias sap.ui.webc.main.types.ToolbarStyling
+ */
+enum ToolbarStyling {
```

Comment:
Some more API dicsussion that's worth taking (@vladitasev  @dobrinyonkov @PetyaMarkovaBogdanova )

`ToolbarStyling` and `ToolbarDesign` are quite similar especially if you aren't a Toolbar expert, but an ordinary app developer.